project/OmnidocBuild.scala

```diff
@@ -195,12 +180,14 @@ object OmnidocBuild {
 
   def compilerReporterSettings = Seq(
     compile / compilerReporter := {
-      new sbt.internal.server.LanguageServerReporter(
+      new sbt.internal.inc.LoggedReporter(
         maxErrors.value,
         streams.value.log,
         foldMappers(sourcePositionMappers.value)
       )
     },
+    tastyFiles := (Compile / compile / tastyFiles).value, // not 100% sure but it seems to work
+    bspReporter := (Compile / compile / bspReporter).value, // same
```

Comment:
Needed to fix (because of sbt upgrade)
```
References to undefined settings:
Omnidoc / javadoc / tastyFiles from Omnidoc / javadoc ((sbt.Defaults.docTaskSettings) Defaults.scala:2123)
Omnidoc / compile / bspReporter from Omnidoc / javadoc ((sbt.Defaults.docTaskSettings) Defaults.scala:2123)
Omnidoc / scaladoc / tastyFiles from Omnidoc / scaladoc ((sbt.Defaults.docTaskSettings) Defaults.scala:2123)
Omnidoc / compile / bspReporter from Omnidoc / scaladoc ((sbt.Defaults.docTaskSettings) Defaults.scala:2123)
```

Actually these two settings does not really matter in this context IMHO, bsp is for build server protocol and tastyFiles for Scala 3, but even if we use Scala 3 here, it should not really matter as well. I keep the comment here in case someone stumples over it and thinks this should be changed.